NSS-Day-Cohort-49/react-nutshell-props-pushers
# Sc weather api

#### Changes Made


1. Modified file `EventCard.js` to include weather forecast data.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout sc-weather-api
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
   > Example: When user does goes to the events page the should be able to see the weather forecast for that event if the event date is in the range of the returned weather data.
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
